testing/sample_framework/src/app_framework.h

```diff
@@ -104,6 +104,12 @@ jobject GetActivity();
 jclass FindClass(JNIEnv* env, jobject activity_object, const char* class_name);
 #endif  // defined(__ANDROID__)
 
+// Returns ture if we run tests that require interaction.
+bool ShouldRunUITests();
```

Comment:
Optional: I'd prefer App Framework not have test-related things in it, and leave those in firebase_test_framework. This is because I intend on using app_framework for sample apps in addition to the tests (at some point in the future).

If you need these methods to be here, that's OK, but maybe name them something non-test related, like "is running in automated UI mode" (or something better).

If you want to leave these as-is, that's ok.